src/Nethermind/Nethermind.Core.Test/Encoding/HeaderDecoderTests.cs

```diff
@@ -156,6 +156,46 @@ public void Can_encode_decode_with_cancun_fields(ulong? blobGasUsed, ulong? exce
         blockHeader.ExcessBlobGas.Should().Be(excessBlobGas);
     }
 
+    [Test]
+    public void Can_encode_decode_with_ValidatorExitRoot()
+    {
+        BlockHeader header = Build.A.BlockHeader
+            .WithTimestamp(ulong.MaxValue)
+            .WithBaseFee(1)
+            .WithWithdrawalsRoot(Keccak.Zero)
+            .WithBlobGasUsed(0)
+            .WithExcessBlobGas(0)
+            .WithParentBeaconBlockRoot(TestItem.KeccakB)
+            .WithDepositsRoot(Keccak.Zero)
+            .WithValidatorExitsRoot(TestItem.KeccakA).TestObject;
```

Comment:
Considering copy-paste error in WithValidatorExists such tests shouldn't pass, if they pass it means they don't test what they should